node/runtime/src/lib.rs

```diff
@@ -872,6 +882,177 @@ impl pallet_contracts::Config for Runtime {
 	type WeightInfo = pallet_contracts::weights::SubstrateWeight<Runtime>;
 }
 
+/// Fixed gas price of `1`.
+pub struct FixedGasPrice;
+
+impl FeeCalculator for FixedGasPrice {
+	fn min_gas_price() -> U256 {
+		// Gas price is always one token per gas.
+		1.into()
+	}
+}
+
+parameter_types! {
+	pub const EthChainId: u64 = 7;
+}
+
+/// Clone of Istanbul config with `create_contract_limit` raised.
+static EVM_CONFIG: EvmConfig = EvmConfig {
+	gas_ext_code: 700,
+	gas_ext_code_hash: 700,
+	gas_balance: 700,
+	gas_sload: 800,
+	gas_sstore_set: 20000,
+	gas_sstore_reset: 5000,
+	refund_sstore_clears: 15000,
+	gas_suicide: 5000,
+	gas_suicide_new_account: 25000,
+	gas_call: 700,
+	gas_expbyte: 50,
+	gas_transaction_create: 53000,
+	gas_transaction_call: 21000,
+	gas_transaction_zero_data: 4,
+	gas_transaction_non_zero_data: 16,
+	sstore_gas_metering: true,
+	sstore_revert_under_stipend: true,
+	err_on_call_with_more_gas: false,
+	empty_considered_exists: false,
+	create_increase_nonce: true,
+	call_l64_after_gas: true,
+	stack_limit: 1024,
+	memory_limit: usize::max_value(),
```

Comment:
seems ominous, we should review these parameters as well and try to compare against ETH as well as other people's impl

Reply:
Yeah Acala has their own version -- we can use that as a reference. However keep in mind these params are me cloning the ISTANBUL_CONFIG from the source repo and only changing the contract creation limit.

Reply:
Here's their version: https://github.com/AcalaNetwork/Acala/blob/master/modules/evm/src/lib.rs#L61

Reply:
Seems basically the same change I made + some gas return changes I'm not sure about.